index.js

```diff
@@ -848,10 +848,7 @@ module.exports = {
 
         // merge responses
         if (key === "responses") {
-          orig[key] = this.mergeObjects(
-            orig[key],
-            toMerge[key],
-          );
+          orig[key] = toMerge[key];
```

Comment:
@davidszanto restore old code.

Need to replace
```js
              // attach common responses
              200: {
                $ref: "#/components/responses/ReturnedData",
```
to
```js
              // attach common responses
              200: {
                "description": "",
                "content": {
                  "application/json": {
                    "schema": {
                      $ref: "#/components/responses/ReturnedData",
                    }
                  }
                }
              },
```
after that change "ReturnedData" to 
```
"ReturnedData": {
            "schema": {
              "oneOf": [
                {
                  "$ref": "#/components/schemas/DbMixinList",
                },
                {
                  "$ref": "#/components/schemas/DbMixinFindList",
                },
                {
                  "$ref": "#/components/schemas/Item",
                },
              ],
            },
          },
```

And so by analogy for 401, 500 etc